dev/clint/src/clint/linter.py

```diff
@@ -117,6 +122,15 @@ def _mlflow_class_name(self, node: ast.ClassDef) -> None:
         if "MLflow" in node.name or "MLFlow" in node.name:
             self._check(node, MLFLOW_CLASS_NAME)
 
+    def _no_timeout(self, node: ast.Call) -> None:
+        if (
+            isinstance(node.func, ast.Attribute)
+            and node.func.attr == "result"
+            and isinstance(node.func.value, ast.Name)
+            and not any(kw.arg == "timeout" for kw in node.keywords)
+        ):
+            self._check(node, NO_TIMEOUT)
```

Comment:
This can lead to false positives.

Reply:
This is NEVER a complete solution. There are more blocking future operations such as `concurrent.futures.as_completed`.

Reply:
I would rather that we have this warning indicator active even if it's not going to mark some false positives. This is a great idea.